Test string, Luka testni controller

lukacukeric77 opened the pull request:
Commit 1, issue#4

Commit:
Test string, Luka testni controller

McNullty reviewed (changes requested):
Nisam dobro definirao zadatak. ali kad sam rekao napravi kontroler mislio sam da kreiraš klasu LukaResorce ili LukaController  koji bi imao endpoint koji si napravio. 

Taj kontroler ćemo koristiti i za druge endpointe. Možeš samo movati endpoint koji si napravio u novui klasu i staviti na nju @RequestMapping("/api/luka") na na endpoint @GetMapping("/test")

Ostalo izgleda dobro

Commit:
Refaktoring:
- LukaResource added as new controller
- AccountResource cleaned
- getTestString added to LukaResource

McNullty reviewed (approved):
Nice
